docs/farming.md

```diff
@@ -19,17 +20,46 @@ You can find these executables in the [Releases](https://github.com/subspace/sub
 
 > This is a ***non-incentivized*** testnet. Meaning there are no rewards in place at this time, and has absolutely no financial benefit to being run at this time.
 
-### A. To Farm With The Network (Online)
+<details><summary> 🖼️ Windows Instructions (Click to Expand)</summary>
```

Comment:
Generally I think we can leave `<details>`, but I'm not really sure we need `<summary>`, we can just make each its own section and remove even more HTML from Markdown that way.